webapp/src/Controller/Jury/ContestController.php

```diff
@@ -504,8 +508,61 @@ public function editAction(Request $request, int $contestId)
                 $problem->setContest($contest);
             }
 
+            // Determine the removed teams, team categories and problems.
+            // Note that we do not send out create / update events for
+            // existing / new problems, teams and team categories. This happens
+            // when someone connects to the event feed (or we have a
+            // dependent event) anyway and adding the code here would
+            // overcomplicate this function
+            $deletedEntities = function(Collection $collection, string $idMethod) {
+                /** @var PersistentCollection $collection */
+                $deletedEntities = [];
+                foreach ($collection->getSnapshot() as $oldEntity) {
+                    $found = false;
+                    foreach ($collection->toArray() as $newEntity) {
+                        $newId = call_user_func([$newEntity, $idMethod]);
+                        $oldId = call_user_func([$oldEntity, $idMethod]);
```

Comment:
For performance would be better to move this line outside the for-loop over `$newEntity`.